boot/makebootable_test.go

```diff
@@ -564,6 +564,56 @@ current_trusted_recovery_boot_assets={"bootx64.efi":["39efae6545f16e39633fbfbef0
 
 	// make sure SealKey was called
 	c.Check(sealKeyCalls, Equals, 1)
+
+	pbc, err := boot.BootChainsFromFile(filepath.Join(dirs.SnapFDEDirUnder(boot.InstallHostWritableDir), "boot-chains"))
```

Comment:
this should just test for the file existence, while the TestSealKeyToModeenv should look at content I think